lib/similarity_utils.py

```diff
@@ -21,3 +22,70 @@ def iou(lista1: List[Any], lista2: List[Any]) -> float:
         intersection = set(lista1).intersection(set(lista2))
         union = set(lista1).union(set(lista2))
         return len(intersection) / len(union)
+
+
+def sort_two_lists(list1: List[Any], list2: List[Any], reverse: bool = True):
+    """metoda sortuje dwie listy równocześnie
+    list1 jest sortowana po wartościach, a kolejność elementów w list2 zależy od sortowania list1
+    reverse = True oznacza kolejność malejącą, a reverse = False oznacza kolejność rosnącą"""
+    zipped_lists = zip(list1, list2)
+    list1, list2 = zip(*sorted(zipped_lists, reverse=reverse))
+    return list1, list2
+
+
+def find_actor(data: pd.DataFrame, actor_id: str) -> List[Any]:
+    """metoda na podstawie ramki danych oraz id aktora, zwraca jego dane z ramki danych"""
+    actor_index = data[data['nconst'] == actor_id].index
+    return data.iloc[actor_index[0]]
```

Comment:
czemu schodzimy to actor_index[0] - jest ich więcej dla jednego nconst?

Reply:
W przypadku bez [0] dosataję cały obiekt pandasowy przypisany do indexu, a nie pojednczą wartość. W przypadku mojego kodu, wydobywa on jedynie liczbę naturalną będącą indeksem w ramce danych